modin/core/execution/dask/implementations/pandas_on_dask/partitioning/partition.py

```diff
@@ -89,20 +86,19 @@ def apply(self, func, *args, **kwargs):
         -----
         The keyword arguments are sent as a dictionary.
         """
-        client = default_client()
         call_queue = self.call_queue + [[func, args, kwargs]]
+        num_returns = 2
```

Comment:
To not introduce such a variable everywhere we should probably declare it a kw arg to deploy.

Reply:
> The only way I can see to improve readability is to change the interface so that this option can be used in this way: num_returns=2.

Yeh, that's what I try to say above, but it also need to change the interface for Ray, and this already looks like a separate change.

Reply:
Ok, let's do it as part of a separate PR.

Reply:
I see the changes for ray in this PR. Are we able to put them in a separate PR?

Reply:
ok